### DISCUSSION TITLE
brew install robotology/formulae/yarp won't work on macbook pro with catalina 10.15.7

### DISCUSSION THREAD

**Discussion by lucarag** [robotology/yarp#2507] [Support - Q&A] [answered]
Hi, 
I followed the instructions listed both here:
http://www.yarp.it/git-master/install_yarp_mac.html#install_mac_Homebrew_YARP

and in this github repository. However, these instruction didn't work, with error messages below:

(base) C-LOSX0UEP3Y2:~ u79992lr$ brew tap homebrew/science
Error: homebrew/science was deprecated. This tap is now empty and all its contents were either deleted or migrated.
(base) C-LOSX0UEP3Y2:~ u79992lr$ brew install robotology/formulae/yarp
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "robotology/formulae/yarp".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.

I'm particularly interested in using yarp in python, and I actually also found this:
https://pypi.org/project/yarp/
but I'm unsure if this is just a wrapper and requires the above instructions to work first. 

Thank you in advance.

(I'm attaching the screenshot with error and instructions)
<img width="1440" alt="Schermata 2021-02-19 alle 13 22 07" src="https://user-images.githubusercontent.com/72132319/108504626-7d7eac00-72b6-11eb-846e-a2d69212d53e.png">

**Comment by Nicogene** [comment, marked as answer]
Hi @lucarag,

The `yarp` formula has been removed since it was not maintained (see https://github.com/robotology/homebrew-formulae/pull/29)

I advise you to follow the instructions for [installing from sources](http://www.yarp.it/git-master/install_yarp_mac.html#install_mac_from_source).

Probably we should remove those instructions from the documentation cc @drdanz @traversaro

**Comment by lucarag** [comment]
Hi @Nicogene,

I followed your advice and was able to complete the installation. I didn't have the chance to use the tool yet, but by typing yarp in the consolle, the output is the same as that in the guide. 

Thanke you again for you help!
